### PR TITLE
[JA] privacy-sandbox translations

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
Adds Japanese translations for
- [privacy-sandbox/feedback/report-2022-q1/](https://developer.chrome.com/docs/privacy-sandbox/feedback/report-2022-q1/)
- [docs/privacy-sandbox/fedcm/](https://developer.chrome.com/docs/privacy-sandbox/fedcm/)
Translated by Alconost

### CODE REVIEW

**Comment by ilyaspiridonov** [comment]
CC: @agektmr

File: site/ja/docs/privacy-sandbox/fedcm/index.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+* [OpenID Connect Front-Channel Logout](https://openid.net/specs/openid-connect-frontchannel-1_0.html)
+* [OpenID Connect Session Management](https://openid.net/specs/openid-connect-session-1_0.html)
+* [iframeベースのバックグラウンドトークン 更新](https://github.com/fedidcg/use-case-library/issues/10)
+* [Iframeベースのログインウィジェット](https://github.com/fedidcg/use-case-library/issues/12)（[Facebook のパーソナル化されたログインボタン](https://developers.facebook.com/docs/facebook-login/web/login-button/)など）
+
+FedCM の最初の目標は、サードパーティ Cookie の段階的廃止が ID フェデレーションに与える影響を減らすことです。上記は、影響を受けると予想される領域のリストです。 その他のユースケースでリストに含まれていないものがある場合は、[貢献とフィードバックの共有](#share-feedback)をお勧めします。
+
+## FedCM の推奨使用対象者 {: #who-uses-fedcm }
+
+以下の**すべて**の条件に該当する場合にのみ、FedCM が役立つことを期待しています。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/fedcm/index.md:55]:
```suggestion
以下の**すべて**の条件に該当する場合にのみ、FedCM が役立つと考えています。
```

File: site/ja/docs/privacy-sandbox/fedcm/index.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+* [リライングパーティへのサインインに、ID プロバイダーを使用する](#sign-in)
+* [リライングパーティからサインアウトする](#sign-out)
+* [認証用のトークンを取り消す](#revoke-tokens)
+
+仕組みについては、[デモ](https://fedcm-rp-demo.glitch.me)を確認してください。
+
+将来的には、以下を含むより多くの機能をサポートしたいと考えています。
+
+*  自動サインイン
+*  承認プロンプト
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/fedcm/index.md:103]:
```suggestion
*  認可プロンプト
```

File: site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md [hunk 3]
```diff
@@ -0,0 +1,723 @@
...
+   </td>
+   <td>Chrome は、そのようなテストを可能にするテスト環境を促進する方法を積極的に模索しています。 <a href="/blog/first-party-sets-sameparty/#how-do-you-get-involved">FPS</a> のローカルテスト用の指示と、<a href="/docs/privacy-sandbox/chips/#try-it-out">CHIPS</a> に関する指示も公開しており、暫定的に使用することができます。
+   </td>
+  </tr>
+  <tr>
+   <td>FedCM
+   </td>
+   <td>明示性
+   </td>
+   <td>ブラウザはフェデレーション ID フローの一部をレンダリングするため、IdP がユーザーに提示したいと思うニュアンスをすべて把握するのは困難であるという懸念
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md:599]:
```suggestion
   <td>ブラウザは ID 連携フローの一部をレンダリングするため、IdP がユーザーに提示したいと思うニュアンスをすべて把握するのは困難であるという懸念
```

File: site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md [hunk 4]
```diff
@@ -0,0 +1,723 @@
...
+   </td>
+   <td>Chrome はこのトレードオフを認識しており、エコシステムと連携して、可能な限り多くの分野をカバーし、可能な限り明示できるようにすることに取り組み続ける意向です。 Chrome は、ブランディングのカスタマイズ（ロゴ、色など）や文字列のカスタマイズ（「～でログイン」ではなく「この記事にアクセス」など）といったアイデアを検討中です。
+   </td>
+  </tr>
+  <tr>
+   <td>FedCM
+   </td>
+   <td>ブラウザの関与
+   </td>
+   <td>ブラウザが以前よりも ID フェデレーションフローに関与しているため、ユーザーがどのウェブサイトに（どの IDP を使って）ログインしているかをより明確に認識することが懸念されます。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md:609]:
```suggestion
   <td>ブラウザが以前よりも ID 連携フローに関与しているため、ユーザーがどのウェブサイトに（どの IDP を使って）ログインしているかをより明確に認識することが懸念されます。
```

File: site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md [hunk 5]
```diff
@@ -0,0 +1,723 @@
...
+   </td>
+   <td>Chrome は、ブラウザがより積極的な役割を果たしていることを認識していますが、フェデレーションをサポートしながら、ブラウザがクロスサイトトラッキングを区別して防止するには、この追加レベルの関与が必要です。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md:611]:
```suggestion
   <td>Chrome は、ブラウザがより積極的な役割を果たしていることを認識していますが、ID 連携をサポートしながら、ブラウザがクロスサイトトラッキングを区別して防止するには、この追加レベルの関与が必要です。 
```

File: site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md [hunk 6]
```diff
@@ -0,0 +1,723 @@
...
+   </td>
+  </tr>
+  <tr>
+   <td>FedCM
+   </td>
+   <td>適用性と相互運用性
+   </td>
+   <td>他のブラウザが FedCM を採用または実装しないことへの懸念。 
+   </td>
+   <td>Chrome は、FedID コミュニティグループでフェデレーションのための共通ソリューションを見つけるために、他のブラウザベンダーとの協力にも努めています。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md:621]:
```suggestion
   <td>Chrome は、FedID コミュニティグループで ID 連携のための共通ソリューションを見つけるために、他のブラウザベンダーとの協力にも努めています。 
```

File: site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md [hunk 7]
```diff
@@ -0,0 +1,723 @@
...
+   </td>
+   <td>Chrome は、エコシステムテストの一環としてこれをさらに調査する予定です。
+   </td>
+  </tr>
+  <tr>
+   <td>FedCM
+   </td>
+   <td>エンタープライズポリシーとユーザーコントロール
+   </td>
+   <td>企業がフェデレーション ID の展開を変更せずに維持できるようにするコントロール（エンタープライズポリシーやユーザー設定など）が提供されるかどうかに関する懸念 再デプロイまたは変更が特別に困難であるフェデレーション ID のオンプレミスデプロイが多いため、IdP の再デプロイを必要とする新しいブラウザ API に対して多くの抵抗があります。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md:639]:
```suggestion
   <td>企業が ID 連携の展開を変更せずに維持できるようにするコントロール（エンタープライズポリシーやユーザー設定など）が提供されるかどうかに関する懸念 再デプロイまたは変更が特別に困難である ID 連携のオンプレミスデプロイが多いため、IdP の再デプロイを必要とする新しいブラウザ API に対して多くの抵抗があります。
```

**Review by agektmr — CHANGES_REQUESTED** [review]
Some changes requested

File: site/ja/docs/privacy-sandbox/fedcm/index.md [hunk 8]
```diff
@@ -0,0 +1,166 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Federated Credential Management API'
+subhead: >
+  プライバシーを保護する ID フェデレーションのためのウェブ API。
```
[the remark this answers is not in the document]
Comment (agektmr) [reply site/ja/docs/privacy-sandbox/fedcm/index.md]: @ilyaspiridonov please see the comment and address other similar cases.

**Comment by ilyaspiridonov** [comment]
Hello @agektmr! My latest commit should accommodate all the requested changes - would you PTAL?

File: site/ja/docs/privacy-sandbox/fedcm/index.md [hunk 9]
```diff
@@ -22,33 +22,33 @@ authors:
...
-過去 10 年間、ID フェデレーションは、使いやすさ（パスワードレスのシングルサインオンなど）、セキュリティ（フィッシング攻撃やクレデンシャルスタッフィング攻撃への耐性の向上など）、サイト単位でのユーザー名とパスワードに比較した信頼性の観点から、ウェブでの認証の水準を上げる上で中心的な役割を果たしてきました。
+過去 10 年間、ID 連携は、使いやすさ（パスワードレスのシングルサインオンなど）、セキュリティ（フィッシング攻撃やクレデンシャルスタッフィング攻撃への耐性の向上など）、サイト単位でのユーザー名とパスワードに比較した信頼性の観点から、ウェブでの認証の水準を上げる上で中心的な役割を果たしてきました。
 
-ID フェデレーションを採り入れることで、RP（リライングパーティ）は IDP（ID プロバイダー）に依存して、ユーザーに新しいユーザー名とパスワードを要求せずにアカウントを提供することが可能となっています。
+ID 連携を採り入れることで、RP（リライングパーティ）は IDP（ID プロバイダー）に依存して、ユーザーに新しいユーザー名とパスワードを要求せずにアカウントを提供することが可能となっています。
 
 
 {% Aside 'key-term' %}
-_ID 連携_ は、信頼できる外部パーティ（_ID プロバイダー_ または IdP）に個人（ユーザーまたはエンティティ）の認証または認可を委任します。 すると ID プロバイダーは、その個人のウェブサイト（_リライングパーティ_ または RP）へのサインインを許可します。
+_ID 連携_は、信頼できる外部パーティ（_ID プロバイダー_ または IdP）に個人（ユーザーまたはエンティティ）の認証または認可を委任します。 すると ID プロバイダーは、その個人のウェブサイト（_リライングパーティ_または RP）へのサインインを許可します。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/fedcm/index.md:31]:
```suggestion
_ID 連携_ は、信頼できる外部パーティ（_ID プロバイダー_ または IdP）に個人（ユーザーまたはエンティティ）の認証または認可を委任します。 すると ID プロバイダーは、その個人のウェブサイト（_リライングパーティ_ または RP）へのサインインを許可します。
```

File: site/ja/docs/privacy-sandbox/fedcm/index.md (hunk 2 above)
Comment (agektmr) [reply site/ja/docs/privacy-sandbox/fedcm/index.md:103]: This regressed.

File: site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md [hunk 10]
```diff
@@ -618,7 +618,7 @@ Chrome は、Public Suffix List など、ウェブプラットフォームで使
    </td>
    <td>他のブラウザが FedCM を採用または実装しないことへの懸念。 
    </td>
-   <td>Chrome は、FedID コミュニティグループで ID 連携のための共通ソリューションを見つけるために、他のブラウザベンダーとの協力にも努めています。 
+   <td>Chrome は、FedID コミュニティグループで連携のための共通ソリューションを見つけるために、他のブラウザベンダーとの協力にも努めています。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md:621]:
```suggestion
   <td>Chrome は、FedID コミュニティグループで ID 連携のための共通ソリューションを見つけるために、他のブラウザベンダーとの協力にも努めています。
```

File: site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md [hunk 11]
```diff
@@ -608,7 +608,7 @@ Chrome は、Public Suffix List など、ウェブプラットフォームで使
    </td>
    <td>ブラウザが以前よりも ID 連携フローに関与しているため、ユーザーがどのウェブサイトに（どの IDP を使って）ログインしているかをより明確に認識することが懸念されます。
    </td>
-   <td>Chrome は、ブラウザがより積極的な役割を果たしていることを認識していますが、ID 連携をサポートしながら、ブラウザがクロスサイトトラッキングを区別して防止するには、この追加レベルの関与が必要です。 
+   <td>Chrome は、ブラウザがより積極的な役割を果たしていることを認識していますが、連携をサポートしながら、ブラウザがクロスサイトトラッキングを区別して防止するには、この追加レベルの関与が必要です。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/feedback/report-2022-q1.md:611]:
```suggestion
   <td>Chrome は、ブラウザがより積極的な役割を果たしていることを認識していますが、ID 連携をサポートしながら、ブラウザがクロスサイトトラッキングを区別して防止するには、この追加レベルの関与が必要です。
```

**Review by agektmr — CHANGES_REQUESTED** [review]
There are quite a few regressions. Please fix them.

File: site/ja/docs/privacy-sandbox/fedcm/index.md (hunk 2 above)
Comment (agektmr) [reply site/ja/docs/privacy-sandbox/fedcm/index.md:103]: 「承認」 needs to be 「認可」

**Comment by ilyaspiridonov** [comment]
@agektmr to make sure we're on the same page: apart from the requested changes that I committed, can you see other regressions)?

**Comment by agektmr** [comment]
No, they are the only remaining change requests.

**Review by agektmr — APPROVED** [review]
LGTM

**Pull request merged**